test/extended/operators/images.go

```diff
@@ -99,20 +99,49 @@ var _ = Describe("[Feature:Platform][Smoke] Managed cluster", func() {
 					if !validImages.Has(container.Image) {
 						invalidPodContainerImages.Insert(fmt.Sprintf("%s/%s/%s image=%s", pod.Namespace, pod.Name, container.Name, container.Image))
 					}
+
 					if container.ImagePullPolicy != v1.PullIfNotPresent {
 						invalidPodContainerImagePullPolicy.Insert(fmt.Sprintf("%s/%s/%s imagePullPolicy=%s", pod.Namespace, pod.Name, container.Name, container.ImagePullPolicy))
 					}
 				}
+				// check if the container's image from the downstream.
+				for j := range pod.Spec.Containers {
+					containerName := pod.Spec.Containers[j].Name
+					commands := []string{
+						"exec",
+						"-n",
+						pod.Namespace,
+						pod.Name,
+						"-c",
+						containerName,
+						"--",
+						"cat",
+						"/etc/redhat-release",
+					}
+					result, err := oc.AsAdmin().RunWithoutGlobalArgs(commands...).Args().Output()
+					if err != nil {
+						e2e.Logf("unable to run command:%v with error: %v", commands, err)
+						continue
+					}
+					e2e.Logf("Image relase info:%s", result)
+					if !strings.Contains(result, "Red Hat Enterprise Linux Server") {
```

Comment:
Not supporting RHEL CoreOS based images ?

Reply:
Not for now, I guess the release images have not based on the `RHCOS` yet. Correct me if I'm wrong. What do you think?@vikaslaad

Reply:
the images this would be checking are all UBI based and I'd expect them to continue to be.

Reply:
Inferring from @smarterclayton - will this continue to be the case if we run OKD CI and with Fedora CoreOS as the base?

Reply:
fedora coreos would be the base for the nodes, not the container images which i would expect to continue to be UBI based.  I'm not aware of any plans for a fedora equivalent of the UBI.

Reply:
@bparees I'm sorry, what're the checkpoints of UBI based images? Do we have any special label when running UBI images?

Reply:
we only run ubi images, there are no other images.

Reply:
@bparees I guess checking the output of `$ cat /etc/redhat-release` is enough for identifying the UBI images, what do you think?

Reply:
@jianzhangbjz i think it's fine for now, if it becomes a problem we can always revisit it.

Reply:
Ok, thanks!